src/pages/api/newsletter/subscribe.ts

```diff
@@ -0,0 +1,41 @@
+import prisma from '@/lib/prisma';
+import { NextResponse } from 'next/server';
+
+export async function POST(request: Request) {
```

Comment:
We will probably need an unsubscribe functionality as well

Reply:
i think we should implement that at the bottom of the newsletter email